docs/concepts/get-offline-features.md

```diff
@@ -0,0 +1,85 @@
+---
+layout: default
+title: Getting Offline Features using Feature Query
+parent: Feathr Concepts
+---
+
+# Getting Offline Features using Feature Query
```

Comment:
I see the concept of `Feature Join` will no longer exist and `Feature Query` will be highlighted. Is there any specific reason for this change?

Reply:
Good q... basically there's no customer facing APIs for "feature join" so I removed that concept from docs as well